core/src/main/java/org/apache/carbondata/core/keygenerator/columnar/ColumnarSplitter.java

```diff
@@ -36,18 +34,16 @@
    *
    * @param keys
    * @return
-   * @throws KeyGenException
    */
-  byte[][] generateAndSplitKey(long[] keys) throws KeyGenException;
+  byte[][] generateAndSplitKey(long[] keys);
```

Comment:
the caller also should handle the throws exception, like: org.apache.carbondata.core.keygenerator.columnar.impl.MultiDimKeyVarLengthVariableSplitGeneratorUnitTest#testGenerateAndSplitKeyAndGetKeyArrayWithActualLogicWithInt

Reply:
This exception is removed

Reply:
I have not check testcase, I think it is ok